packages/env/lib/build-docker-compose-config.js

```diff
@@ -70,6 +71,24 @@ function getMounts(
 	];
 }
 
+/**
+ * Gets information about the host user so we can implement ownership parity.
+ *
+ * @return {Object} The host user's name, uid, and gid.
+ */
+function getHostUser() {
+	const hostUser = os.userInfo();
+	const uid = ( hostUser.uid === -1 ? 0 : hostUser.uid ).toString();
+	const gid = ( hostUser.gid === -1 ? 0 : hostUser.gid ).toString();
```

Comment:
As per [Node's documentation](https://nodejs.org/api/os.html#osuserinfooptions), on Windows the `uid` and `gid` is `-1`. I think this only happens with a `process.platform` of `win32`. If you're using WSL you will realistically be running commands _inside_ the container, so, this is more of an edge-case where they have Node installed on Windows.

The question _I_ have is whether or not this breaks anything. I'm curious about what kind of permission magic takes place when going mounting from Windows -> Docker instead of WSL -> Docker. If it works anything like macOS, I would wager that having everything owned by `root` doesn't matter.